packages/dataviews/src/components/dataviews-view-config/index.tsx

```diff
@@ -214,8 +214,13 @@ function SortDirectionControl() {
 }
 
 function ItemsPerPageControl() {
-	const { view, perPageSizes, onChangeView } = useContext( DataViewsContext );
-	if ( perPageSizes.length < 2 || perPageSizes.length > 6 ) {
+	const { view, config, onChangeView } = useContext( DataViewsContext );
+	if (
+		! config ||
+		! config.perPageSizes ||
```

Comment:
Is this is a change in behaviour? Previously `perPageSizes` had some default values but these are no longer set? `[ 10, 20, 50, 100 ]`